docs/prepping-local-system-runbook-linux.md

```diff
@@ -228,176 +217,167 @@ This gcloud configuration is called [default]. You can create additional configu
+12. To view information about your Cloud SDK installation and the active SDK configuration, enter:
 
     `gcloud info`
 
-    These instructions were sourced from the Google document, [Quickstart for Linux](https://cloud.google.com/sdk/docs/quickstart-linux), which includes additional information.
+These instructions were sourced from the Google document, [Quickstart for Linux](https://cloud.google.com/sdk/docs/quickstart-linux), which includes additional information.
 
-### Installing Docker
+### For EKS only: installing the eksctl utility
 
-For Linux command line users, you can follow these steps to install Docker Community Edition (CE) for the first time on a new host machine. For these instructions, you need to set up the Docker repository. Afterward, you can install and update Docker from the repository.
+Pega recommends deploying your EKS cluster for your Pega Platform deployment using the AWS `eksctl` command line utility for creating and managing clusters on Amazon EKS.
 
-**SET UP THE REPOSITORY**
+1. To download and extract the latest `eksctl` binary from the AWS site, from your home directory, enter:
 
-1. Update the apt package index:
+   `$ curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp`
 
-    `$ sudo apt-get update`
+2. To move the extracted binary to /usr/local/bin, which allows the command to be run without the "sudo" prefix, enter:
 
-2. Install packages to allow apt to use a repository over HTTPS:
+   `$ sudo mv /tmp/eksctl /usr/local/bin`
 
-    `$ sudo apt-get install apt-transport-https ca-certificates curl software-properties-common`
+3. To verify the `eksctl` version, enter:
 
-3. Add Docker’s official GPG key:
+   ```bash
+   $ eksctl version
+   0.17.0
+   ```
 
-    `$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add –`
+These instructions were sourced from the AWS document, [Install eksctl](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html), which includes additional information about how to use the 'eksctl' utility in combination with the 'kubectl' command-line utility, which you should have already installed earlier in this document.
 
-4. Verify that you now have the key with the fingerprint `9DC8 5822 9FC7 DD38
-    854A E2D8 8D81 803C 0EBF CD88` by searching for the last 8 characters of the
-    fingerprint.
+### For EKS only: installing AWS CLI version 2
 
-    `$ sudo apt-key fingerprint 0EBFCD88`
+Install the AWS CLI utility with a packaging tool such as apt-get.
 
-    The command should return:
+`$ sudo apt-get install awscli`
+
+To confirm the installation or upgrade, enter:
 
 ```bash
-pub 4096R/0EBFCD88 2017-02-22
-9DC8 5822 9FC7 DD38 854A E2D8 8D81 803C 0EBF CD88
-uid Docker Release (CE deb) <docker@docker.com>
-sub 4096R/F273FCD8 2017-02-22
+$ aws --version
+aws-cli/2.0.10 Python/3.7.3 Linux/5.3.0-51-generic botocore/2.0.0dev14
```

Comment:
This did not work for me until I set a SAML profile for aws.  I assume there is a specific prcess that each customer would follow for this.

Reply:
I added details into the linux doc